extensions/amp-list/amp-list.md

```diff
@@ -331,7 +331,7 @@ An `<amp-list-load-more>` element with the `load-more-button` attribute, which s
 ##### Example:
 
 ```html
-<amp-list load-more="auto" src="https://www.load.more.example.com/" width="400" height="800">
+<amp-list load-more="manual" src="https://www.load.more.example.com/" width="400" height="800">
```

Comment:
This was added to address feedback that only `manual` lists will ever see a load-more button.